CoffeeChat/Controller/InterestsViewController.swift

```diff
@@ -147,7 +130,33 @@ class InterestsViewController: UIViewController {
                 switch result {
                 case .value(let response):
                     if response.success {
-                        // TODO: Fix interest model in backend before using to populate screen
+                        self.selectedInterests = response.data.interests.map {
+                            // TODO: Fix interest model in backend before using to populate screen
```

Comment:
is the model going to reference all the different icons in the assets folder for the interests?